integration/nwo/template/configtx_template.go

```diff
@@ -226,16 +226,6 @@ Profiles:{{ range .Profiles }}
         Endorsement:
           Type: ImplicitMeta
           Rule: "MAJORITY Endorsement"
-    {{- if .Consortium }}
-    Consortium: {{ .Consortium }}
-    {{- else }}
```

Comment:
get rid of the if else, and the part in the else, but keep assigning the consortium name unconditionally. Thus you need to reverse the removal of `Consortium` from the profile